.script/SolutionValidations/solutionValidator.ts

```diff
@@ -123,12 +33,20 @@ let CheckOptions = {
     // Callback function to handle errors during execution
     onExecError: async (e: any, filePath: string) => {
         console.log(`Solution Validation Failed. File path: ${filePath}. Error message: ${e.message}`);
+        if (e instanceof MainTemplateDomainVerticalValidationError) {
+            logger.logError("Please refer link https://github.com/MicrosoftDocs/azure-docs/blob/main/articles/sentinel/sentinel-solutions.md?msclkid=9a240b52b11411ec99ae6736bd089c4a#categories-for-microsoft-sentinel-out-of-the-box-content-and-solutions for valid Domains and Verticals.");
```

Comment:
Can we also log actual error message thrown from method?

Reply:
logging the original or custom error message to console using  console.log(`Solution Validation Failed. File path: ${filePath}. Error message: ${e.message}`);